style.css

```diff
@@ -0,0 +1,19 @@
+@media (max-width: 600px) {
+    #navbar {
+      padding: 0rem 1rem;
+    }
```

Comment:
please use flex with a flex-direction row it will make the nav bars nicer to look. so please use the following code and modify it.
      display: flex;
      flex-direction: row;
      gap: 3px;